.github/dependabot.yml

```diff
@@ -42,3 +42,10 @@ updates:
     - dependency-name: "com.google.api.grpc:grpc-*"
     - dependency-name: "com.google.http-client:*"
     - dependency-name: "com.google.apis:google-api-services-*"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      # Allow all official github actions
```

Comment:
Optional - might be helpful to specifically call out the approval limitation for future readers